src/backend/joanie/tests/core/test_models_enrollment.py

```diff
@@ -135,7 +135,7 @@ def test_models_enrollment_forbid_for_non_listed_course_run(self):
         )
 
         with self.assertRaises(ValidationError) as context:
-            factories.EnrollmentFactory(course_run=course_run)
+            factories.EnrollmentFactory(course_run=course_run, is_active=True)
```

Comment:
Just a pin as it is related to my comment about the clean method of Enrollment model.